protoc-gen-kotlin/lib/src/commonMain/kotlin/pbandk/gen/CodeGenerator.kt

```diff
@@ -26,7 +30,35 @@ open class CodeGenerator(
         fn().also { indent = indent.dropLast(4) }
     }
 
-    protected fun writeType(type: File.Type, parentType: String? = null) = when (type) {
+    private fun writeExtension(field: File.Field) {
+        when (field) {
+            is File.Field.Numbered -> {
+                val localType = when (field.type) {
+                    File.Field.Type.MESSAGE -> ((field as File.Field.Numbered.Standard).localTypeName ?: field.kotlinLocalTypeName)?.substringAfter(".")
+                    else -> field.type.standardTypeName
+                }
+                val fieldExtendee = field.extendee!!.trimStart('.').replace("google.protobuf", "pbandk.wkt")
+                line(
+                    "val $fieldExtendee.${field.name}: ${localType}? get() ="
+                )
+                line("  getExtension(${file.packageName}.${field.name})")
```

Comment:
Instead of embedding spaces in the line, this should be:

```kt
line("val $fieldExtendee.${field.name}: ${localType}? get() =").indented {
    line("getExtension(${file.packageName}.${field.name})")
}
```